content/applications/inventory_and_mrp/purchase/products/uom.rst

```diff
@@ -70,11 +70,11 @@ and name the category.
    :align: center
    :alt: Create a new units of measure category in Odoo Purchase
 
-The next step is to create the two units of measures. To do so, go to :menuselection:`Configuration
---> Units of Measure`.
+The next step is to create the two units of measures. To do so, click into the Unit of Measure
+category. Under the :guilabel:`Units of Measure` tab, click :guilabel:`Add a line`.
 
 First, create the unit of measure used as the reference point for converting to other units of
-measure inside the category by clicking on *Create*. Name the unit and select the units of measure
+measure inside the category. Name the unit and select the units of measure
```

Comment:
Early line break (move category to line 77)

Reply:
Suggestion implemented ✅